{
  "bug_location": "Unknown",
  "severity": 3,
  "categories": [
    "Unknown"
  ],
  "root_cause": "N/A - Analysis failed",
  "issue_number": 7909,
  "title": "cases::test_merge::test_node_merge_write_data_to_source_region_after_merging failed"
}